### PR TITLE
Update social share texts

### PR DESCRIPTION
**Pull request by alicoding**
re: https://github.com/MozillaFoundation/encryption_campaign/issues/179

### CODE REVIEW

**Comment by alicoding** [comment]
@alanmoo R?

File: pages/encrypt/index.jsx [hunk 1]
```diff
@@ -21,14 +21,14 @@ module.exports = React.createClass({
...
           <meta property="og:site_name" content="Mozilla Advocacy - Encryption"/>
-          <meta property="og:url" content="https://advocacy.mozilla.org/encrypt/?video=2"/>
+          <meta property="og:url" content="https://advocacy.mozilla.org/encrypt/"/>
           <meta property="og:description" content="I just watched a great video about encryption. Watch it yourself and learn more about how encryption works in our everyday lives."/>
           <meta property="og:image" content="https://advocacy.mozilla.org/assets/Encryption-social-05-07.png" />
 
           <meta name="twitter:card" content="summary_large_image" />
           <meta name="twitter:site" content="@mozilla" />
-          <meta name="twitter:title" content="Meet Encryption" />
-          <meta name="twitter:description" content="I just learned how encryption works in our everyday lives. Check it out at mzl.la/encrypt via @mozilla #encrypt" />
```
Comment (alanmoo) [inline pages/encrypt/index.jsx]: Where's the hashtag go? And I think we are moving from "Privacy lets you be you" for the title, though "Meet Encryption" was a late night guess for me. @pauljohnson?

Comment (alicoding) [inline pages/encrypt/index.jsx]: This is not for sharing part except for the twitter card.

Comment (alicoding) [inline pages/encrypt/index.jsx]: Oh and the Meet Encryption was my mistake I will put that back

File: pages/encrypt/index.jsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ module.exports = React.createClass({
           <meta property="og:type" content="website" />
           <meta property="og:title" content="Check out this awesome video"/>
           <meta property="og:site_name" content="Mozilla Advocacy - Encryption"/>
-          <meta property="og:url" content="https://advocacy.mozilla.org/encrypt/?video=2"/>
```
Comment (alicoding) [inline pages/encrypt/index.jsx]: Why ?video2 is even here?

Comment (alanmoo) [inline pages/encrypt/index.jsx]: Future proofing/permalinking

Comment (alicoding) [inline pages/encrypt/index.jsx]: I don't think you do it here... but here instead https://github.com/mozilla/advocacy.mozilla.org/pull/103/files#diff-f3157f69c013e71be91db4856735b839L76 if you want to link to that specifically. This was meant for something else.

**Comment by alanmoo** [comment]
LGTM

**Pull request merged**